packages/trackerless-network/test/end-to-end/proxy-connections.test.ts

```diff
@@ -205,7 +205,7 @@ describe('Proxy connections', () => {
         await waitForCondition(() => 
             proxiedNode.hasProxyConnection(streamPartId, getNodeIdFromPeerDescriptor(proxyNodeDescriptor1), ProxyDirection.SUBSCRIBE)
         , 25000)
-        expect(proxyNode1.hasProxyConnection(streamPartId, proxiedPeerId.toKey() as unknown as NodeID, ProxyDirection.SUBSCRIBE)).toBe(false)
+        expect(proxyNode1.hasProxyConnection(streamPartId, proxiedPeerId.toKey() as unknown as NodeID, ProxyDirection.SUBSCRIBE)).toBe(true)
```

Comment:
Was this test assertion wrong before? Seems like now it properly works? (i.e. the proxy connection is recovered one the proxy node comes online?)

Reply:
Yeah there seemed to be a bug with the test.